tests/ClientTest.php

```diff
@@ -2528,9 +2516,7 @@ public function testOurSerializer($after_creation)
                     ],
                 ]
             );
-            $events = $client->getSentEvents();
-            $event = array_pop($events);
-            $this->assertEquals(Dummy_Serializer::$default_value, $event['extra']['foo']);
+            $serializer->__phpunit_verify();
```

Comment:
We must be sure serialize is invoked on every accessible type. I create foreach block for this and only for this purpose.